quick_start.adoc

```diff
@@ -1,22 +1,142 @@
 [#getting-started]
 = Getting started
 
-* <<manage-clusters,Manage clusters>>
-* <<manage-applications,Manage applications>>
-* <<manage-security,Manage security>>
-* <<observability-quick, Observe clusters>>
+* <<introduction,Introduction>>
+* <<create-and-manage-clusters,Create clusters>>
+* <<manifestwork-example, ManifestWork example>>
 
 [#introduction]
 == Introduction
 
 Coming: Architecture and intro.
 
-Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../install/requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your hub cluster, you can proceed with link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online].
+Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your multicluster engine for Kubernetes cluster, you can proceed with link:../install_connected.adoc#installing-while-connected-online[Installing while connected online].
 
+[#create-and-manage-clusters]
+== Create and manage clusters
 
-[#manage-clusters]
-== Manage clusters
+You are now ready to create and import clusters.
+From your multicluster engine for Kubernetes cluster, you can create clusters from other Kubernetes services to manage.
```

Comment:
(Disclaimer: I am reading this phrase in diff-mode, not the entire text as a whole, so this may be off point.)

I don't understand the point of the "from other Kubernetes services" phrase here in general, and the particular emphasis it seems to provide on the "other services" aspects.  If we are talking about creating clusters via MCE/Hive, then the only kind of clusters we can create are OCP ones, not the non-OCP *KS type clusters that seem to be implied by the "other services" wording.  And whether or not the clusters created are on same service or other service doesn't seem germane, in general.

I would suggest this revised to end "...you can create other Openshift Container Platform clusters to manage."